From 1c1039dfd57164f5602c5425925c52a343e04ec0 Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Fri, 10 May 2013 16:16:53 +0800
Subject: [PDK MR2 Temp] backport dbus support

BZ: 117199

Revert "Remove DBUS from make files"
This reverts commit b1f1013321e1f64fd43a3e9fa008ac38845b773f.

This patch restores DBUS temorarily as Google removes it pdk

Change-Id: Ieaf68789ef00b2777f7983864798ebe10d6ce6e0
---
 core/legacy_prebuilts.mk |    1 +
 core/pathmap.mk          |    1 +
 target/product/base.mk   |    2 ++
 3 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/core/legacy_prebuilts.mk b/core/legacy_prebuilts.mk
index f767644..d0ec83d 100644
--- a/core/legacy_prebuilts.mk
+++ b/core/legacy_prebuilts.mk
@@ -35,6 +35,7 @@ GRANDFATHERED_ALL_PREBUILT := \
 	chat-ril \
 	cpcap-key.kl \
 	data \
+	dbus.conf \
 	dev \
 	egl.cfg \
 	firmware_error.565 \
diff --git a/core/pathmap.mk b/core/pathmap.mk
index b76e0ed..2442889 100644
--- a/core/pathmap.mk
+++ b/core/pathmap.mk
@@ -30,6 +30,7 @@ pathmap_INCL := \
     bootloader:bootable/bootloader/legacy/include \
     camera:system/media/camera/include \
     corecg:external/skia/include/core \
+    dbus:external/dbus \
     frameworks-base:frameworks/base/include \
     frameworks-native:frameworks/native/include \
     graphics:external/skia/include/core \
diff --git a/target/product/base.mk b/target/product/base.mk
index 129fb2a..08f1bcd 100644
--- a/target/product/base.mk
+++ b/target/product/base.mk
@@ -26,6 +26,7 @@ PRODUCT_PACKAGES += \
     bmgr \
     bugreport \
     content \
+    dbus-daemon \
     dhcpcd \
     dhcpcd-run-hooks \
     dnsmasq \
@@ -43,6 +44,7 @@ PRODUCT_PACKAGES += \
     libcamera_client \
     libcameraservice \
     libchromium_net \
+    libdbus \
     libdl \
     libdrm1 \
     libdrm1_jni \
-- 
1.7.4.1

